environments/development.docker-compose.yaml

```diff
@@ -121,7 +121,6 @@ services:
       - "--state-dir=/app/state"
       - "--ocluster-pool=./capnp-secrets/pool-linux.cap"
       - "--docker-cpu=${OCAML_BENCH_DOCKER_CPU?required}"
-      - "--multicore-repositories=${OCAML_BENCH_MULTICORE_REPOSITORIES}"
```

Comment:
Yes so the issue is that the worker has a limited pool of isolated CPUs available (= number of ocluster jobs they can handle at the same time) and we currently have no way of using more than one CPU on a per repository basis: The worker doesn't know the name of the repo to toggle that behaviour like before AND even if it did, this would lead to the worker accepting more ocluster jobs than it can handle right now (we don't have a way to communicate "this job costs 2 cpu and I only have one left").

The plan is that the pipeline will schedule the multiple cores benchmarks to different workers: there will be 1cpu workers (the default) and 2cpu workers etc. We will use the repository configuration to specify which workers it should use for each repository: to enable multicore benchmarks, we would associate a repository with the name of a multicore worker.

Now in order to run a multicore worker with this PR, we can use ranges:

```
OCAML_BENCH_DOCKER_CPU=2-3,4-5,6-7
```

This worker will use 2 CPU per benchmarks, with a maximum of 3 concurrent jobs.

Of course we should be careful in our CPU lists if we run two workers on the same machine so that their CPUs don't overlap!

Reply:
okay, this sounds good! Thanks!